NEWS.md

```diff
@@ -1,6 +1,10 @@
 # meetupr 0.1.1
 * Added `get_event_rsvps()` function.  Contribution by Michael Beigelmacher: https://github.com/rladies/meetupr/pull/19
-
+* Added automatic rate limiting based on the response headers.
```

Comment:
The current version number is [0.2.3](https://github.com/rladies/meetupr/commit/09442970c7c279816736f425dcba0458e333a723).  

I think we've been forgetting to update the NEWS.md file...oops.

Can you increment the version number to and move your updates in NEWS.md to a new section?  Since this is a pretty major update (yay!), maybe we should increment to 0.3.0?

Reply:
FYI I created a ticket to go back and fill in the missing sections of NEWS.md (we can do that later) https://github.com/rladies/meetupr/issues/102